core/chains/evm/client/client.go

```diff
@@ -29,7 +29,11 @@ const queryTimeout = 10 * time.Second
 
 // Client is the interface used to interact with an ethereum node.
 type Client interface {
-	txmgrtypes.Client[*big.Int, evmtypes.Nonce, common.Address, types.Block, common.Hash, types.Transaction, common.Hash, types.Receipt, types.Log, ethereum.FilterQuery]
+	commontypes.Client[*big.Int, evmtypes.Nonce, common.Address, types.Block, common.Hash, types.Transaction, common.Hash, types.Receipt, types.Log, ethereum.FilterQuery]
+	commontypes.Accounts[common.Address, evmtypes.Nonce]
```

Comment:
Since Client interface already also includes Transactions, Blocks, Events, can we skip mentioning them here?

Reply:
yup agreed with removing the duplicate method declaration for clarity.